### PR TITLE
move current Ruby code snippets directory to V4

### PR DESCRIPTION
**Pull request by jchen293**
Move current Ruby code snippets directory to V4
Add Ruby v5 code snippets for the address before moving to the rest

### CODE REVIEW

File: official/docs/ruby/v4/addresses/verify.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  street2: 'FLOOR 5',
+  city: 'SAN FRANCISCO',
+  state: 'CA',
+  zip: '94104',
+  country: 'US',
+  company: 'EasyPost',
+  phone: '415-123-4567',
+)
+
+address.verify
```
Comment (Justintime50) [inline official/docs/ruby/v4/addresses/verify.rb:16]: issue: there are no instance methods

**Review by Justintime50 — CHANGES_REQUESTED** [review]

File: official/docs/ruby/v4/addresses/verify.rb (hunk 1 above)
Comment (jchen293) [reply official/docs/ruby/v4/addresses/verify.rb:16]: oops, how did i miss this

Comment (Justintime50) [reply official/docs/ruby/v4/addresses/verify.rb:16]: Doesn't look like this was pushed though?

Comment (jchen293) [reply official/docs/ruby/v4/addresses/verify.rb:16]: The one you marked is on the `v4` code snippet, this only applies to the `current` code example😉

Comment (Justintime50) [reply official/docs/ruby/v4/addresses/verify.rb:16]: Reading is hard

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**